data-raw/northern-ireland-lad-summary-metrics.R

```diff
@@ -0,0 +1,132 @@
+library(tidyverse)
+library(IMD)
+library(geographr)
+library(compositr)
+library(sf)
+library(ggridges)
+
+ltla <-
+  boundaries_ltla21 |>
+  st_drop_geometry() |>
+  filter(str_detect(ltla21_code, "^N"))
+
+lookup_northern_ireland_ltla <-
+  lookup_ltla_ltla |>
+  filter(str_detect(ltla21_code, "^N"))
+
+# ---- IMD score ----
+# Higher extent = more deprived /
+# Higher rank (calculated here) = more deprived
+imd <- 
+  imd_northern_ireland_lad |>
+  select(ltla21_code = lad_code, imd_score = Extent) |>
+  mutate(number = rank(imd_score)) |>
+  select(-imd_score) |>
+  mutate(variable = "Index of Multiple \nDeprivation rank", .after = ltla21_code) |>
+  mutate(percent = NA, .after = number)
+
+# ---- ONS Health Index ----
+
+# An official Health Index for Northern Ireland does not exists. Use the BRC Resilience Index version
+
+# Higher score = worse health
+# Higher rank (calculated here) = worse health
+health_index_raw <- read_csv(
+  "https://raw.githubusercontent.com/britishredcrosssociety/resilience-index/main/data/vulnerability/health-inequalities/northern-ireland/index-unweighted-all-indicators.csv"
+)
+
+health_index <- 
+  health_index_raw |>
+  select(ltla21_code = lad_code, number = health_inequalities_composite_rank) |>
+  mutate(percent = NA) |>
+  mutate(variable = "Health Index \nrank") |>
+  relocate(variable, .after = ltla21_code)
+
+# ---- % Left-behind areas ----
+
+#The left-behind areas come from the community needs index (loaded from the IMD package)
```

Comment:
Add space at the beginning: Change `#The` to `# The`

Reply:
If you are not already using the `styler` package. I recommend doing so and running it on each file you create. For me, it caught this small inconsistency.

Reply:
space added